src/components/BorrowerProfile/DetailsTabs.vue

```diff
@@ -50,6 +50,7 @@
 						@show-definition="showDefinition"
 					/>
 					<repayment-schedule
+						v-if="loan.status !== 'fundraising' || loan.amonimizationLevel !== 'none'"
```

Comment:
Need to fix spelling for `anonymizationLevel` and update the check to be `!== 'full'`. Most loans are set to `none` so this current check would exclude almost all loans.

In the near future we can get more sophisticated to check for `public` too.